### DISCUSSION TITLE
mpv在播放一些 iso 格式的 dvd 文件时，会出现时间轴时长残缺的情况，有什么脚本可以完美播放 iso 文件吗？

### DISCUSSION THREAD

**Discussion by wathper** [hooke007/mpv_PlayKit#470] [Q&A 问答]
各个脚本收集的github仓库中没有找到可用的脚本，开发组也不打算解决 ios 文件的兼容问题。

目前只能开启 cache=yes 预读文件来加载读取不到的时间轴时长。

但是效果也不好。

请问有可以完美打开 ios 加载完整的菜单及时间轴的脚本吗？
或者其他成熟的解决方案？

**Comment by hooke007** [comment]
是ISO，无。

**Reply by wathper** [reply]
好吧，打错字了

**Reply by Ericxisuo** [reply]
ISO 目前比较好的还是vlc
